src/units/attack_type.hpp

```diff
@@ -84,6 +85,7 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 	std::vector<std::pair<t_string, t_string>> special_tooltips(boost::dynamic_bitset<>* active_list = nullptr) const;
 	std::string weapon_specials() const;
 	std::string weapon_specials_value(const std::set<std::string> checking_tags) const;
+	std::pair<unit_alignments::type, bool> specials_alignment() const;
```

Comment:
/**
 * Returns the ToD to use when calculating damage, including effects from [attack_alignment] specials.
 * If no such specials are active, it returns the unit's alignment.
 * 
 * The boolean returned indicates whether the ToD bonus for the unit's alignment should be applied (by
 * the caller) in addition to the ToD from the attack; this is based on [attack_alignment]cumulative=.
 */